ext_localconf.php

```diff
@@ -1,11 +1,16 @@
 <?php
 
+use Sng\Additionalscheduler\Command\AdditionalSchedulerCommandController;
+use Sng\Additionalscheduler\Utils;
+use TYPO3\CMS\Core\Utility\ExtensionManagementUtility;
+use Vendor\ExtKey\Command\SimpleCommandController;
```

Comment:
in ext_localconf and ext_tables prefer dont use import NS, use full reference class here

Reply:
ok. (Why ?)

Reply:
I think its because all this files are concatenate in one, it result in some "use" statement in the middle of a file wich is not very "psr friendly" and readable.

Reply:
Yes indeed ! I remember I've already met that problem.